src/components/Map.vue

```diff
@@ -2,7 +2,15 @@
   <section class="mb-4">
     <l-map class="map" :class="stac.type" ref="leaflet" @ready="init()">
       <l-control-fullscreen />
-      <l-tile-layer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" :options="mapOptions" />
```

Comment:
I think we should go with a v-if , v-else here.

Reply:
Updated to use v-if, v-else. It is not working to look for and use sys:targets if it exists. Otherwise, it uses the standard open street map.